pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -478,8 +475,8 @@ in with passthru; stdenv.mkDerivation (finalAttrs: {
   optionalString enableNoSemanticInterposition ''
     export CFLAGS_NODIST="-fno-semantic-interposition"
   '' + optionalString (stdenv.buildPlatform.isFreeBSD && stdenv.hostPlatform.isFreeBSD) ''
```

Comment:
Any reason not to make this unconditional?

Reply:
I'm worried that other code paths for other operating systems won't take kindly to the empty uname -r substitute.

Reply:
Well, uname isn't going to work on them anyway, right?  And we don't have any better value for them.

Reply:
I was referring to other unix-likes like NetBSD, OpenBSD, and SunOS. I'm totally amenable to simply doing this and seeing what happens, though!